src/command/KeyBindingManager.js

```diff
@@ -1084,7 +1096,7 @@ define(function (require, exports, module) {
     function _getDisplayKey(key) {
         var displayKey = "",
             match = key ? key.match(/(Up|Down|Left|Right|\-)$/i) : null;
-        if (match) {
+        if (match && !/Page(Up|Down)/.test(key)) {
             displayKey = key.substr(0, match.index) + _displayKeyMap[match[0].toLowerCase()];
```

Comment:
No, we can't match the start of text since `key` string may have some modifier keys as in `Cmd-Home`.